docs/source/tools/traffic_vault_util.rst

```diff
@@ -28,7 +28,7 @@ The ``traffic_vault_util`` tool - located at :file:`tools/traffic_vault_util.go`
 
 Usage
 =====
-``traffic_vault_util [--dry_run] --vault_ip IP --vault_action ACTION [--vault_user USER] [--vault_password PASSWD] [--vault_port PORT]``
+``traffic_vault_util [--dry_run] --vault_ip IP --vault_action ACTION [--vault_user USER] [--vault_password PASSWD] [--vault_port PORT] [--skip_tls] [--server_name SERVER_NAME]``
```

Comment:
nit: in most places we just call this `--insecure` instead of `--skip_tls`

Reply:
Dig it, done.